src/webextension/lib/util.js

```diff
@@ -8,3 +10,13 @@ export const makePageAction = tab => {
   browser.pageAction.show(tab.id);
   logger.log(`Created pageAction for ${tab.id}`);
 };
+
+// If being run in the WebExtension context, send a message to the SDK add-on.
+// Otherwise use the regular logger.
+export const sendMessage = (type, payload) => {
```

Comment:
@rpl Paired with the [webpack dev server](https://github.com/chuckharmston/pulse/blob/b41c0de4d2985a63c5f9b6c261fa1d2b1b37fcef/README.md#survey), this is how I'm handling the unclear context. Not particularly inventive, but it gets the job done.